src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
+  const pickTab = (tab: Tab) => {
+    tab.id !== selectedTab.id && onTabSelected(tab)
```

Comment:
![image](https://github.com/mate-academy/react_tabs/assets/84296041/5cb5e4c5-09ed-46fd-bd7e-fe8e57991a05)


```suggestion
   if (tab.id !== selectedTab.id) {
    onTabSelected(tab)
   }
```